templates/playlist/playlists.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<div class="container-fluid">
+  <div class="row">
+
+        <div class = "col-md-3">
+            <form method = 'POST' id = "CreatePlaylist">
```

Comment:
CONSISTENCY!

your format for tag attributes `attribute="<value>"` # fuck the space and you use double quote.

BE CONSISTENT